ml-algorithms/src/main/java/org/opensearch/ml/engine/MLEngine.java

```diff
@@ -136,7 +139,7 @@ private static void validateMLInput(Input input) {
         }
     }
 
-    private static void validateInput(Input input) {
+    private void validateInput(Input input) {
```

Comment:
I assume we are removing `static` to make it easy in writing unit test, right?

Reply:
Yes, will easier for both unit test and integration test.